src/components/footer.tsx

```
@@ -1,97 +1,158 @@
-import Link from "next/link";
-import { ExternalLink, MapPin, Mail } from "lucide-react";
-import { mainNavItems, footerLinks } from "@/config/navigation";
-import { TYPOGRAPHY } from "@/constants/typography";
-import { GAP } from "@/constants/spacing";
-import { cn } from "@/lib/utils";
-import { BrandName } from "@/components/brand-name";
+import Link from 'next/link';
+import { 
+  Brain, FileText, Bell, Users,
+  Calendar, LayoutGrid, Building, MessageSquare, 
+  Mail, Infinity, BookOpen, Cloud, ExternalLink
```

Biome (1.9.4)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)